test/e2e/configs.go

```diff
@@ -241,3 +247,16 @@ invalid:
  - rule2: ManyInstancesDown
    expr: job:up:avg{job="node"} < 0.5
 `
+
+const validYamlWithInvalidRulesYamlTpl = `
```

Comment:
Also, had to add a new invalid case. 
So if YAML is completely invalid, then Observatorium API won't be able to parse it and respond with a `500` status code, as is covered by the `write-invalid-rules` test case.

In the new `write-valid-yaml-with-invalid-rules` case, the YAML is valid and API is able to parse it and sends it to rules backend, after which validation fails and it responds with `400` status code.

Open to suggestions on how this can be handled better. 🙂

Reply:
I think this is correct, in the end user will end up with `4xx`, understanding that an invalid rule has been provided.